### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by mp46241** [willswire/unifi-ddns#79] [Q&A] [answered]
i am new to this, i cannot get the deploy to workers to take my account info/api token. i have created a token with access to edit workers but it keeps saying, "Account ID or API Token are not valid. Check your Account ID is correct and API Token has permissions to Edit Workers"

**Comment by FrAllard** [comment, marked as answer]
I used the "Edit Cloudflare Workers" templace
![image](https://github.com/user-attachments/assets/94b985d4-8759-4001-9373-345898133a1a)
I setup the "Zone Resources" according to what I wanted, and the result was this :
![image](https://github.com/user-attachments/assets/234c8c62-8e7e-495b-b6e7-41a8f17baf5d)
and it worked for me.
At least the deployment form was satisfied and showed a Deployed at the top, but didn't deploy in my CF account in the Workers & Pages section. I'm here to figure this out and I saw your thread for help.
